docs/csharp/programming-guide/main-and-command-args/top-level-statements.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Top-level statements - C# Programming Guide"
+description: Learn about top-level statements in C# 9 and later.
+ms.date: 03/08/2021
+helpviewer_keywords:
+  - "C# language, top-level statements"
+  - "C# language, Main method"
+---
+# Top-level statements (C# Programming Guide)
+
+Starting in C# 9, you don't have to explicitly include a `Main` method in a console application project. Instead, you can use the *top-level statements* feature to minimize the code you have to write. In this case, the compiler implicitly provides a class and `Main` method entry point for the application.
```

Comment:
Consider applying similar change to what @BillWagner suggested here. This aligns with the *IsImplicitlyDeclared* compiler API being false, and also aligns with the fact that these have `[CompilerGenerated]` attribute.

```suggestion
Starting in C# 9, you don't have to explicitly include a `Main` method in a console application project. Instead, you can use the *top-level statements* feature to minimize the code you have to write. In this case, the compiler generates a class and `Main` method entry point for the application.
```